tuning/data/setup_dataprocessor.py

```diff
@@ -118,7 +118,7 @@ def process_dataargs(
         kwargs = {
             "fn_kwargs": fn_kwargs,
             "batched": False,
-            "remove_columns": [JSON_INPUT_KEY, JSON_OUTPUT_KEY],
+            "remove_columns": "all",
```

Comment:
In current implementation, dataset just have these [columns](https://github.com/dushyantbehl/fms-hf-tuning/blob/163fe34fca481ae06549a73d2785f8dde4d39c88/tuning/data/data_handlers.py#L52) `input_ids,  labels", attention_mask`, hence every other columns in the dataset needed to be removed.